### PR TITLE
Fix documentation to reduce warnings

### PR DESCRIPTION
**Pull request by alkino**
This patch try to reduce the number of warnings and error when building documentation.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -121,6 +121,7 @@ def _config_exe(exe_name):
     zip_safe=False,
     setup_requires=[
         "jinja2>=2.9.3",
+        "jupyter-client==6.1.12",
```
Comment (pramodk) [inline setup.py:124]: In principle, building NMODL doesn't require Jupyter. Once we will upload source wheels to pypi.org, building via pip will also install whole Jupyter stuff. So we should abvoid it from setup.py?

**Review by pramodk — CHANGES_REQUESTED** [review]
I have one comment/question about setup.py

I assume you have checked the style/format of generated oxygen doc.

File: setup.py (hunk 1 above)
Comment (alkino) [reply setup.py:124]: jupyter is already a depends of nmodl for building notebooks in documentation

Comment (pramodk) [reply setup.py:124]: discussed offline - jupyter is already installed. This merely installs different jupyter-client version due to bug.

**Review by pramodk — APPROVED** [review]

**Pull request merged**